edx_toggles/toggles.py

```diff
@@ -0,0 +1,44 @@
+from abc import ABC
+
+
+from django.conf import settings
+
+
+class BaseToggle(ABC):
+    def __init__(self, name, default=False):
+        self.name = name
+        self.default = default
+
+    def is_enabled(self):
+        raise NotImplementedError
+
+
+class ListedTogglesMixin:
```

Comment:
> Should this be called something like InstanceTrackingMixin or InstanceListMixin?

I admit that `ListedTogglesMixin` is not a very good name. I'll go with `ClassInstancesMixin`, if you agree.

> This could be used by WaffleFlag, etc., but BaseToggle (or BaseSettingToggle) cannot.

Why not? I expect to inherit from this class once waffle flags have been migrated.